steps/step-fauxpasapp/0.1.1/step.yml

```diff
@@ -0,0 +1,115 @@
+title: Fauxpasapp Step
+summary: Step for Fauxpasapp
+description: |-
+  This is a Step for Fauxpasapp.
+
+  [Faux Pas](http://fauxpasapp.com)
+
+  This is not an officially supported plugin
+website: https://github.com/mackoj/step-fauxpasapp
+source_code_url: https://github.com/mackoj/step-fauxpasapp
+support_url: https://github.com/mackoj/step-fauxpasapp/issues
+published_at: 2016-03-18T17:45:29.629193545+01:00
+source:
+  git: https://github.com/mackoj/step-fauxpasapp.git
+  commit: fc2ed2154405b1b1e097b6bd19a2841f67f4b74e
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- script
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- fauxpas_license_type: personal
+  opts:
+    is_dont_change_value: true
+    is_expand: false
```

Comment:
Most of the times is_expand: true is better then false, because it allows you to specify the input value with env var, otherwise just direct values are allowed.
is_expand: false is usefull if your direct value contains "$" letter.
Please set it to `true`.